floris/simulation/grid.py

```diff
@@ -69,6 +72,8 @@ class Grid(ABC):
     wind_speeds: NDArrayFloat = field(converter=floris_array_converter)
     time_series: bool = field()
 
+    _xc_rot: float = field(init=False)
```

Comment:
Are these safe to delete? I only see them written to but not read from.

Reply:
Good catch, I think you are right. These may be fine to delete. It's generally worth thinking through how the heterogeneous map is interpolated and applied in the wind-aligned frame that FLORIS does its calculations in. Though, I think the example in #579 shows that the variable `_xc_rot` is probably not needed.

Reply:
Ok great I've removed them